test/e2e/utils_test.go

```diff
@@ -1101,6 +1123,28 @@ func updateCRPWithTolerations(tolerations []placementv1beta1.Toleration) {
 	}, eventuallyDuration, eventuallyInterval).Should(Succeed(), "Failed to update cluster resource placement with tolerations %s", crpName)
 }
 
+func buildClusterNamesWithoutTaints(taintClusterNames []string) []string {
+	taintClusterNameMap := make(map[string]bool)
+	for i := range taintClusterNames {
+		taintClusterNameMap[taintClusterNames[i]] = true
+	}
+	var noTaintClustersNames []string
+	for i := range allMemberClusterNames {
+		if !taintClusterNameMap[allMemberClusterNames[i]] {
+			noTaintClustersNames = append(noTaintClustersNames, allMemberClusterNames[i])
+		}
+	}
+	return noTaintClustersNames
+}
+
+func buildClustersFromNames(clusterNames []string) []*framework.Cluster {
```

Comment:
Hi Arvind! Is this really necessary? The cluster array can be composed directly in a pretty easy manner; this utility feels like an overkill.

Reply:
In most (if not all) references to this utility, it's just returning a list of cluster objects for looping; we only have three functioning clusters in the E2E environments and their respective cluster objects are already top-level variables that can be directly referenced.